experimental/framework/tinylicious-client/src/FluidStaticTinylicious.ts

```diff
@@ -0,0 +1,99 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Container } from "@fluidframework/container-loader";
+import {
+    ContainerConfig,
+    DOProviderContainerRuntimeFactory,
+    FluidContainer,
+} from "@fluid-experimental/fluid-static";
+import {
+    ITinyliciousServiceConfig,
+    TinyliciousService,
+} from "./tinyliciousService";
+
+/**
+ * FluidTinyliciousInstance provides the ability to have a Fluid object backed by a Tinylicious service
+ */
+export class FluidTinyliciousInstance {
+    private readonly containerService: TinyliciousService;
+
+    public constructor(port?: number) {
```

Comment:
Let's have the constructor take an object with optional port. It will make this more extensible in the future.

Reply:
Updated! This also lines up well with OdspClient, which will be taking in a config object. [This](https://github.com/microsoft/FluidFramework/pull/5838/files#diff-ca79fa4af55271157ecfcfbc8fe3a66008833fa106efb320fbb92e629a86b169) holds the two interface definitions that are specific to Tinylicious.

The three separate configs then are now called: ITinyliciousConnectionConfig, ITinyliciousContainerConfig, and ContainerSchema

I*ConnectionConfig is the naming format for the interface defining the config used for service initialization and used for all containers created/loaded by the app
I*ContainerConfig is the naming format for the interface that is service-specific and can be uniquely set for each container that is created/loaded by the app

To avoid confusion amongst ContainerConfigs, the old ContainerConfig name from FluidStatic is now renamed to ContainerSchema since it is defining how the data will be organized within the container itself. This interfaces is uniform for all services but its values may be unique for each container.